google/auth/transport/_mtls_helper.py

```diff
@@ -405,3 +406,43 @@ def client_cert_callback():
 
     # Then dump the decrypted key bytes
     return crypto.dump_privatekey(crypto.FILETYPE_PEM, pkey)
+
+def check_use_client_cert():
+  """Returns the effective value of use_client_cert to be used.
+
+  Returns:
+      str:
+          A boolean indicating if client certificate should be used.
```

Comment:
Convert to a boolean instead of checking strings everywhere?

Will be a bit cleaner.